packages/nimble-components/src/table-column/extensions/tests/fractional-width-column.spec.ts

```diff
@@ -0,0 +1,62 @@
+/* eslint-disable max-classes-per-file */
+import { ViewTemplate, ElementStyles, html } from '@microsoft/fast-element';
+import { fixture, Fixture } from '../../../utilities/tests/fixture';
+import type { TableCellState } from '../../base/types';
+import { TableColumn } from '../../base';
+import {
+    FractionalWidthColumn,
+    fractionalWidthColumn
+} from '../fractional-width-column';
+
+class TestTableColumnBase extends TableColumn {
+    public cellTemplate: ViewTemplate<TableCellState> = html``;
+    public cellStyles?: ElementStyles | undefined;
+    public cellRecordFieldNames: readonly string[] = [];
+}
+
+class TestTableColumnBaseMixin extends TestTableColumnBase {}
+// eslint-disable-next-line @typescript-eslint/no-empty-interface
+interface TestTableColumnBaseMixin extends FractionalWidthColumn {}
+
+class TestTableColumn extends fractionalWidthColumn(TestTableColumnBaseMixin) {}
+
+const composedTestTableColumn = TestTableColumn.compose({
+    baseName: 'fractional-width-test-table-column'
+});
+
+// prettier-ignore
+async function setup(): Promise<Fixture<TestTableColumn>> {
+    return fixture(composedTestTableColumn());
+}
+
+describe('FractionalWidthColumn', () => {
```

Comment:
I think there should be a few more tests here that verify removing the `fractional-width` or `min-pixel-width` attribute does the correct thing.

Reply:
Added tests.